sdk/storage/Azure.Storage.Files.Shares/perf/Azure.Storage.Files.Shares.Perf/Scenarios/DownloadFile.cs

```diff
@@ -0,0 +1,151 @@
+﻿//Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.Storage.Files.Shares.Perf.Scenarios
+{
+    /// <summary>
+    /// The performance test scenario focused on downloading files from the Files Shares storage.
+    /// </summary>
+    /// <seealso cref="Azure.Test.Perf.PerfTest{SizeOptions}" />
+    public sealed class DownloadFile : PerfTest<SizeOptions>
+    {
+        /// <summary>
+        /// Name of the directory client.
+        /// </summary>
+        private string DirectoryClientName { get; set; }
+
+        /// <summary>
+        /// Name of the file client.
+        /// </summary>
+        private string FileClientName { get; set; }
```

Comment:
Is this property used?

Reply:
Yes, in `Run()` and `RunAsync()` methods.

Reply:
My bad. Somehow missed that.

What's the reason we use properties instead of fields for private members?

Reply:
In general, yes I would use fields.
Here, i simply wanted the 2 test classes to look similar to be able to easily highlight the differences. The same properties are internal in the other class.

I'll do some refactoring as per the other comment and these should then only exist in one class.